frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './Team.css'
+import {
+    Card, CardHeader, CardTitle, CardText, CardDeck, CardBody, CardColumns
+} from 'reactstrap';
+
+//Displays the team's information using cards. The cards are contained in a Card deck and arranged by card columns.
+// Cards have header and text components with slightly varying colors. Other styling handled by imported CSS file.
+//The entire functional component is contained in a div tag that uses a container to add whitespace.
+//Before the card deck is a header.
+function TeamCardDeck (props) {
+    return (
+
+        <div className = "container"> {/*Container tag adds whitespace to both sides of element*/}
+            <h1 style = {{textAlign:"center"}}>Meet The Team</h1> {/*header*/}
```

Comment:
I think here you  can use the class "text-center" to achieve the same thing instead of using inline styling.